opencensus/exporters/stats/stackdriver/internal/stackdriver_exporter.cc

```diff
@@ -31,7 +31,8 @@ namespace {
 
 constexpr char kGoogleStackdriverStatsAddress[] = "monitoring.googleapis.com";
 constexpr char kProjectIdPrefix[] = "projects/";
-constexpr int kTimeSeriesBatchSize = 3;
+// Stackdriver limits a single CreateTimeSeries request to 200 series.
```

Comment:
If possible, please add URL which states this.

Reply:
I am not aware of one--the only place I have seen 200 is the error response when you try to upload more.